worlds/stardew_valley/test/__init__.py

```diff
@@ -306,6 +306,12 @@ def create_item(self, item: str) -> StardewItem:
             self.multiworld.worlds[self.player].total_progression_items -= 1
         return created_item
 
+    def remove_one_by_name(self, item: str) -> None:
+        self.remove(self.create_item(item))
+
+    def reset_collection(self):
```

Comment:
Shouldn't this be called `reset_state` instead?

Reply:
The thinking was that it's used in the same test where you collect stuff, like by calling `self.collect_everything()`, so resetting the collection kind of make sense? How about `reset_collected`? I don't like `reset_state` because the other method interacting with the collection state are called `collect` and do not talk about `state`.

Reply:
`reset_collection_state()`?